android/src/main/java/cl/json/social/ShareIntent.java

```diff
@@ -130,9 +138,17 @@ protected static String urlEncode(String param) {
 
         return extraIntents;
     }
+
     protected void openIntentChooser() throws ActivityNotFoundException {
-        Intent chooser = Intent.createChooser(this.getIntent(), this.chooserTitle);
-        chooser.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+        Intent chooser;
+        IntentSender intentSender = null;
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
Should use the static isSupported() method . you have on TargetChosenReceiver to avoid propagating build versions everywhere

Reply:
I have updated the code to use isSupported() in place of build versions.